src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingNamedTypeSymbol.cs

```diff
@@ -375,6 +375,8 @@ public sealed override bool AreLocalsZeroed
             get { throw ExceptionUtilities.Unreachable; }
         }
 
+        internal override FileIdentifier? AssociatedFileIdentifier => null;
```

Comment:
>internal override FileIdentifier? AssociatedFileIdentifier => null; [](http://example.com/codeflow?start=8&length=67)

This doesn't look right. This should return a value that we would get from metadata if the underlying compilation was emitted.

Reply:
Makes sense. I will try and see if there is any useful test coverage that we could add here.